lib/db/schema/patch-067-068.sql

```diff
@@ -0,0 +1,165 @@
+ALTER TABLE `unverifiedTokens`
```

Comment:
is this file missing the boilerplate for `SET NAMES`, via https://github.com/mozilla/fxa-auth-db-mysql/blob/master/scripts/migration.sh#L35 ?